accounts/models.py

```diff
@@ -0,0 +1,114 @@
+from django.db import models
+from django.contrib.auth.models import (
+	 BaseUserManager, AbstractBaseUser
+	)
+# Create your models here.
+
+class UserManager(BaseUserManager):
+
+    def create_user(self, email, password=None,is_active=True, is_staff=False, is_admin=False):
+        """
+        Creates and saves a User with the given email and password.
+        """
+        if not email:
+            raise ValueError('Users must have an email address')
+        if not password:
+        	raise ValueError('Users must have an password')
+
+        user = self.model(
+            email=self.normalize_email(email),
+        )
+
+        user.set_password(password)
+        user.active = is_active
+        user.staff = is_staff
+        user.admin = is_admin
+        user.save(using=self._db)
+        return user
+
+    def create_staffuser(self, email, password):
+        """
+        Creates and saves a staff user with the given email and password.
+        """
+        user = self.create_user(
+            email,
+            password=password,
+            is_staff=True
+        )
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(self, email, password):
+        """
+        Creates and saves a superuser with the given email and password.
+        """
+        user = self.create_user(
+            email,
+            password=password,
+            is_staff=True,
+            is_admin=True
+        )
+        
+        user.save(using=self._db)
+        return user
+
+
+class User(AbstractBaseUser):
+
+	email = models.EmailField(max_length=255,unique=True)
+	active = models.BooleanField(default=True)
+	staff = models.BooleanField(default=False)  # a admin user; non super-user
```

Comment:
why are we still commenting like this? why do I have to fix same errors/reviews for every pr?!!